From 8045a638b22b286ceebac2d5d2c9b6010d5c8f95 Mon Sep 17 00:00:00 2001
From: Ioan Sucan <isucan@gmail.com>
Date: Sat, 15 Feb 2014 19:28:42 -0800
Subject: [PATCH] add install rule for package.xml

---
 CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8d3f920..8a1b7f3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -62,3 +62,6 @@ endif()
 SET_DIRECTORY_PROPERTIES(PROPERTIES
   ADDITIONAL_MAKE_CLEAN_FILES ${CMAKE_BINARY_DIR}/console_bridge-config.cmake
   ADDITIONAL_MAKE_CLEAN_FILES ${CMAKE_BINARY_DIR}/console_bridge.pc)
+
+# Install catkin package.xml
+install(FILES package.xml DESTINATION share/console_bridge)
-- 
1.9.0.rc1.175.g0b1dcb5

