From a2665525f8d2bbcc92bdc6ca49980ba681f1da31 Mon Sep 17 00:00:00 2001
From: Ioan Sucan <isucan@gmail.com>
Date: Sat, 15 Feb 2014 19:28:42 -0800
Subject: [PATCH] add install rule for package.xml

---
 CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ab0ad99..2ce7884 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -58,3 +58,6 @@ endif()
 SET_DIRECTORY_PROPERTIES(PROPERTIES
   ADDITIONAL_MAKE_CLEAN_FILES ${CMAKE_BINARY_DIR}/console_bridge-config.cmake
   ADDITIONAL_MAKE_CLEAN_FILES ${CMAKE_BINARY_DIR}/console_bridge.pc)
+
+# Install catkin package.xml
+install(FILES package.xml DESTINATION share/console_bridge)
-- 
1.8.3.2

