From 30dc18a48ae99d2f531a301fdf7e89fea0e07cb7 Mon Sep 17 00:00:00 2001
From: Dirk Thomas <dthomas@osrfoundation.org>
Date: Wed, 12 Feb 2014 10:08:43 -0800
Subject: [PATCH] add install rule for package.xml

---
 CMakeLists.txt |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4de4210..0cc6c28 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -46,3 +46,6 @@ if (NOT MSVC)
   configure_file("${pkg_conf_file}.in" "${pkg_conf_file}" @ONLY)
   install(FILES ${pkg_conf_file} DESTINATION lib/pkgconfig/ COMPONENT pkgconfig)
 endif()
+
+# Install catkin package.xml
+install(FILES package.xml DESTINATION share/console_bridge)
-- 
1.7.10.4

